spec/series_search_spec.rb

```diff
@@ -127,7 +127,7 @@
     it "everything search, phrase" do
       resp = solr_resp_ids_from_query  '"Studies in Modern Poetry"'
       expect(resp.size).to be >= 15
-      expect(resp).to include(['5709847', '4075051', '3865171', '10338326', '7146913'])
+      expect(resp).to include(['5709847', '4075051', '3865171', '10727485', '588517'])
```

Comment:
This one I’m not sure about. The spec is for series search. Ckey 7146913 has the phrase “Studies in Modern Poetry” in the series field. Ckey 588517 is not in that series, but has the phrase in its title. All the others ckeys in this list are also in the series. But the test is not doing a series search but rather an everything search, so maybe it’s fine to replace ckey 7146913 with 588517?